ee/modules/110-istio/templates/control-plane/validatingwebhook-global.yaml

```diff
@@ -0,0 +1,46 @@
+# Helm cannot deploy the manifests because it stumbles on this webhook, so if isGlobalRevisionIstiodReady is false,
```

Comment:
```suggestion
# Helm cannot deploy the DestinationRule and PeerAuthentication manifests for tlsMode parameter because it stumbles on this webhook, so if isGlobalRevisionIstiodReady is false,
```